packages/nodes-base/nodes/Oracle/Sql/actions/versionDescription.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable n8n-nodes-base/node-filename-against-convention */
+
+import { NodeConnectionTypes, type INodeTypeDescription } from 'n8n-workflow';
+
+import * as database from './database/Database.resource';
+
+export const versionDescription: INodeTypeDescription = {
+	displayName: 'Oracle Database',
+	name: 'oracleDatabase',
+	icon: 'file:oracle.svg',
+	group: ['input'],
+	version: [1],
+	subtitle: '={{ $parameter["operation"] }}',
+	description: 'Get, add and update data in Oracle database',
+	defaults: {
+		name: 'Oracle Database',
+	},
+	inputs: [NodeConnectionTypes.Main],
+	outputs: ['main'],
```

Comment:
Better to use `NodeConnectionTypes.Main` here too
```suggestion
	outputs: [NodeConnectionTypes.Main],
```